src/contracts/pods/EigenPodManager.sol

```diff
@@ -44,6 +44,21 @@ contract EigenPodManager is
         _;
     }
 
+    /**
+    * @notice Modifier to ensure that the deneb fork timestamp is not set
+    * @dev Note that this function is only ever meant to be called twice.  First, it will be called to set the timestamp
+    * to type(uint64).max, and then it will be called to set the timestamp to the actual timestamp
+    */
+    modifier denebForkEnabled(uint64 newDenebForkTimestamp) {
+        require(newDenebForkTimestamp != 0, "EigenPodManager.denebForkEnabled: cannot set newDenebForkTimestamp to 0");
+        if(newDenebForkTimestamp == type(uint64).max){
+            require(denebForkTimestamp == 0, "EigenPodManager.denebForkEnabled: denebForkTimestamp must not be set yet");
+        } else {
+            require(denebForkTimestamp ==type(uint64).max, "EigenPodManager.denebForkEnabled: Deneb fork timestamp cannot be set");
+        }
+        _;
+    }
```

Comment:
since this modifier is only used once, can't it just be a 'require' statement inside the `setDenebForkTimestamp` function?
I'm also confused about the code structure here -- right now the contract owner can transition `denebForkTimestamp`:
`0 -> type(uint64).max`
or `type(uint64).max -> nonzero`
but not `0 -> nonzero` (which is actually what we'd need on Goerli, at least).

I was imagining just doing:
```
require(newDenebForkTimestamp != 0, "EigenPodManager.denebForkEnabled: cannot set newDenebForkTimestamp to 0");
require(denebForkTimestamp == 0 || denebForkTimestamp == type(uint64).max)
```
is there a reason we can't just do that instead? I think it'd be easier to understand + reason through.

Reply:
to clarify: I think what's there would _technically_ meet the need. I just find it confusing and am seeing a particular case in which it seems to raise the complexity unnecessarily, so would prefer a simpler + clearer implementation.

Reply:
My idea is that the modifer is going to be used exactly twice, once to set it to unit64.max and once to set it to the actual timestamp.  Maybe a simpler way to do it is just make it callable twice with an incrementor.

Reply:
perhaps we're talking past each other a bit here.
by "used once" I meant "appear in exactly one place"

I get your intention, but was trying to illustrate my own reasoning here.

I'm going to just mark this as acceptable; as I said above it technically meets the needs here.

Reply:
oh oh I totally misunderstood,  I will remove the modifier and add it as logic

Reply:
+1, let's add requires to the method this is used in. This doesn't need to be a modifier.